critiquebrainz/frontend/templates/review/modify/artist.html

```diff
@@ -1,11 +1,11 @@
 {% if entity is not defined %}
-  {% set entity = review.entity_id | entity_details(type=entity_type) %}
+  {% set entity = entity_id | entity_details(type=entity_type) %}
 {% endif %}
 <div class="col-md-12">
   <dl class="dl-horizontal">
     <dt>{{ _('Artist') }}</dt>
     <dd>
-      {{ entity['name'] | default(_('[Unknown artist]')) }}
+      <a href="{{ url_for('artist.entity', mbid=entity_id) }}" target="_blank">{{entity['name'] | default(_('[Unknown artist]')) }}</a>
```

Comment:
I'm not sure why we have this `default` value in these descriptions. In this file, I believe that we would always have a name, so I recommend that we remove it from all of these files

Reply:
I found out why this `default` is set, it's in case the item no longer exists in musicbrainz (CB-400). However in this specific situation I don't think there's a problem with removing this. Thanks!